### PR TITLE
Double time for ExplicitIndexTest, cleanup deprecated things

### PR DESCRIPTION
**Pull request by MishaDemianenko**

### CODE REVIEW

**Comment by burqen** [comment]
@MishaDemianenko From your change set I don't see how the test failure in `TestWhiteListJavaWrapper` can be related. But this is the only time this test has failed and it also fails locally. I don't see this test failure on `3.4 HEAD`, but I see it after rebasing your branch on `3.4 HEAD`. So my conclusion is that there is some more work for you to do here ;)

**Comment by MishaDemianenko** [comment]
@burqen yeah, his past is hard and future is unclear, let me see whats wrong with it! thx )

**Comment by MishaDemianenko** [comment]
@burqen here it is

File: community/server/src/main/java/org/neo4j/server/scripting/UserScriptClassWhiteList.java [hunk 1]
```diff
@@ -64,6 +64,8 @@ private UserScriptClassWhiteList()
         safe.add( Node.class.getName() );
         safe.add( Relationship.class.getName() );
         safe.add( RelationshipType.class.getName() );
+        // can't list antonymous inner classes, using instance to get name of its class
+        safe.add( RelationshipType.withName( "classSource" ).getClass().getName() );
```
Comment (burqen) [inline community/server/src/main/java/org/neo4j/server/scripting/UserScriptClassWhiteList.java]: "antonymous"?

**Review by burqen — APPROVED** [review]
Not sure I understand the comment. Do with it as you please.

File: community/server/src/main/java/org/neo4j/server/scripting/UserScriptClassWhiteList.java (hunk 1 above)
Comment (MishaDemianenko) [reply community/server/src/main/java/org/neo4j/server/scripting/UserScriptClassWhiteList.java]: hehe, for a moment he was special )

**Pull request merged**